libs/datamodel/core/src/transform/ast_to_dml/db/types.rs

```diff
@@ -11,8 +11,9 @@ use std::{
 pub(super) fn resolve_types(ctx: &mut Context<'_>) {
     for (top_id, top) in ctx.db.ast.iter_tops() {
         match (top_id, top) {
-            (ast::TopId::Alias(alias_id), ast::Top::Type(type_alias)) => visit_type_alias(alias_id, type_alias, ctx),
+            (ast::TopId::Alias(alias_id), ast::Top::Alias(alias)) => visit_alias(alias_id, alias, ctx),
             (ast::TopId::Model(model_id), ast::Top::Model(model)) => visit_model(model_id, model, ctx),
+            (ast::TopId::Type(_type_id), ast::Top::Type(_type_def)) => todo!(),
```

Comment:
If we want to merge this, we should start with a `()` instead of a panic, because anybody with a `type` declaration in their schema will hit the panic.